jobs/apply_scryfall_migrations.js

```diff
@@ -0,0 +1,138 @@
+const fetch = require("node-fetch");
+const carddb = require('../serverjs/carddb');
+const Cube = require('../dynamo/models/cube');
+const _ = require('lodash');
+
+const lastRun = new Date("2000-01-01");
```

Comment:
Should this eventually be stored in the database so that the script can be run automatically?